examples/pipe/interlock_test.c

```diff
@@ -177,21 +194,24 @@ int interlock_test(void)
   ret = pthread_join(writerid, &value);
   if (ret != 0)
     {
-      fprintf(stderr, "interlock_test: pthread_join failed, error=%d\n", ret);
+      fprintf(stderr, \
+        "interlock_test: pthread_join failed, error=%d\n", ret);
       ret = 6;
       goto errout_with_fifo;
     }
   else
     {
-      printf("interlock_test: writer returned %d\n", (int)value);
-      if (value != (void*)0)
+      printf("interlock_test: writer returned %p\n", value);
+      p = (void *) 0;
```

Comment:
```suggestion
      p = (void *)0;
```

Reply:
Fixed in https://github.com/apache/incubator-nuttx-apps/pull/1340